src/plugins/intel_gna/frontend/scale_factor_calc.hpp

```diff
@@ -233,14 +233,14 @@ static bool requantizeInput(InferenceEngine::CNNLayerPtr input, float newOutputS
         auto maxSF = CalculateScaleFactorFromStats(levels, quantDataForInputLayer->_dst_quant.GetMinValues().front(),
             quantDataForInputLayer->_dst_quant.GetMaxValues().front());
         if (newOutputScale > maxSF) {
-            gnalog() << layer->name << ": Scale factor " << newOutputScale << " is too large. The maximum scale factor: "
+            GnaLog::LogDebug() << layer->name << ": Scale factor " << newOutputScale << " is too large. The maximum scale factor: "
                 << maxSF << " levels=" << levels << " min=" << quantDataForInputLayer->_dst_quant.GetMinValues().front()
                 << " max=" << quantDataForInputLayer->_dst_quant.GetMaxValues().front() << "\n";
             return false;
         }
     }
     if (info.isActivation() || info.isConst()) {
-        gnawarn() << "[WARNING] requantize " << layer->name
+        GnaLog::LogErr() << "[WARNING] requantize " << layer->name
```

Comment:
For me warning should be in case when something risk or not proper, but we can still provide functionality.
Error should be rather reserved for the stuff we cannot properly execute task.

Reply:
Yes, you are right here, I've just replaced all warn with LogErr().
Will review them and fix.